cmd/sealer/cmd/cluster/run.go

```diff
@@ -218,6 +222,38 @@ func runWithClusterfile(clusterFile string, runFlags *types.RunFlags) error {
 	return createNewCluster(imageEngine, cf, imageSpec, runFlags.Mode)
 }
 
+func getClusterInstallInfo(labels map[string]string) clusterruntime.InstallInfo {
+	cri := labels["cluster.alpha.sealer.io/container-runtime-type"]
+	if cri == "" {
+		cri = "docker"
+	}
+	clusterRuntimeType := labels["cluster.alpha.sealer.io/cluster-runtime-type"]
+	if clusterRuntimeType == "" {
+		clusterRuntimeType = "kubernetes"
+	}
+	logrus.Infof("The cri is %s, crt is %s\n", cri, clusterRuntimeType)
+	return clusterruntime.InstallInfo{
+		ContainerRuntimeType: cri,
+		ClusterRuntimeType:   clusterRuntimeType,
+	}
+}
+
+func getClusterConfPath(labels map[string]string) string {
+	clusterRuntimeType := labels["cluster.alpha.sealer.io/cluster-runtime-type"]
+	if clusterRuntimeType == "" {
+		clusterRuntimeType = "/etc/kubernetes"
```

Comment:
clusterRuntimeType = "kubernetes" ?, or just return

Reply:
That be my fault, I will correct it.